### ISSUE TITLE
Error while setting up Pytorch based DL model as serverless function for auto annotation

### ISSUE THREAD

**Issue by NamalJayasuriya** [opencv/cvat#6895]
### My actions before raising this issue
I'm trying to use auto annotation for a project with my already trained DL model.
I deployed CVAT v2.7.1 using the github sourcecode.
My CVAT setup is working fine and I followed the Serverless turotial (https://opencv.github.io/cvat/docs/manual/advanced/serverless-tutorial/#dl-model-as-a-serverless-function) from your documentation. I wanted to use with GPU , however I tried with cpu-first to make sure that everything is working fine.

Projects under Openvino (omz-public-yolo-v3,  omz-public-mask-rcnn) works fine and I was able to try the auto annotation. tf-faster-rcnn example also worked well.
But, anything under pytorch did not worked for me. I tried already existing retinanet_r101 in github @v2.7.1. Also tried with the codes given at the above mentioned serverless tutorial.

## steps before error
All the containers and docker images remain due to previous tries are  are killed and run `docker system prune` as well.
Make sure sure CVAT is working fine with other models.
`serverless/deploy_cpu.sh serverless/tensorflow/faster_rcnn_inception_v2_coco/` worked well and tried auto annotation.
Then I stopped faster rcnn using nuclio dashboard at localhost:8070, also checked running containers (screen shot attached).
![image](https://github.com/opencv/cvat/assets/8591605/7f2f57c2-a207-478f-88b4-98e14f115eaf)

Then run `serverless/deploy_cpu.sh serverless/pytorch/facebookresearch/detectron2/retinanet_r101/` and got following errors. Two screen shots attached showing start and end of error output as it is too long.
![image](https://github.com/opencv/cvat/assets/8591605/680aa62c-355b-4c53-a78a-d209c089a048)
![image](https://github.com/opencv/cvat/assets/8591605/d853a8ee-e784-46b7-8091-3a9362eda95e)

I also tried the code given in tutorial mentioned above as well and ended up with something similar.
It will great if you can support to resolve the errors.

### My Environment
- Ubuntu 22.04 6.2.0-33-generic
- Docker version 24.0.6, build ed223bc

**Comment by alan30408** [comment]
@NamalJayasuriya, I also got this issue. Is there any update here?
